closed/src/java.base/share/classes/openj9/internal/criu/CRIUSECProvider.java

```diff
@@ -46,6 +46,7 @@ public CRIUSECProvider() {
 
         String[] aliases = new String[] { "SHA",
                                           "SHA1",
+                                          "SHA-1",
```

Comment:
This seems a reasonable addition, but the reference in `SHA1PRNG` should use the primary name (`"SHA"`), not the added alias.

Reply:
I take back my 'reasonable' comment. `"SHA-1"` is the primary name of that service - the alias should be redundant.
I do agree that "SHA-1" is a better way to refer to the algorithm, but the references should be consistent.

Reply:
done